### DISCUSSION TITLE
Two ideas to improve

### DISCUSSION THREAD

**Discussion by espec** [debloper/piosk#58] [Ideas]
The variables for the scheduler script could be added to the dashboard screen along with the URLs.

Add a script to hide the cursor if inactive. I have seen many ways to do this, but I'm unsure which is best, and those I tried didn't work.

**Comment by debloper** [comment]
Parameterizing options in the dashboard is already planned for future release, thanks for adding support to it.

Discussion on hiding mouse cursor: https://github.com/debloper/piosk/issues/23
